.github/workflows/build.yml

```diff
@@ -115,6 +115,7 @@ jobs:
         - '{ "tfm": "net6.0", "emsdk": "2.0.23" }' # eol: 2024-11-12
         - '{ "tfm": "net7.0", "emsdk": "3.1.12" }' # eol: 2024-05-14
```

Comment:
@ericsink Have you considered removing unsupported binaries?

Reply:
Not really.  Perhaps I should.

But usually "unsupported" != "nobody will complain when it goes away".

Reply:
lol yes, always the dilemma.

Reply:
These assets are particularly painful to keep around because new ones have to be added every release. This is because the emscripten binary format isn't stable and older versions aren't guaranteed to work with newer versions. This is different from iOS and Android where you can just pick a minimum version and live with one binary for years.

Reply:
So you're saying that a (purely hypothetical) maintainer who only wants to avoid complaints might actually get MORE complaints by not removing these old files?

Reply:
lol, eventually once the package size becomes unreasonable, yes.